### DISCUSSION TITLE
SO 2.3 esxi install error 0%

### DISCUSSION THREAD

**Discussion by Triller-purple** [Security-Onion-Solutions/securityonion#3627] [Unsupported Versions]
Hi all,

I'm installing  Security Onion 2.3 on a ESXI 7 host via an .iso. After completing all the questions as part of the installation, it gets stuck at "Running Initial Configuration Steps" 0% and never moves. I've tried 

I've tried evaluation and distribution and to no avail. 

I have 4 cpus allocated to it.
200 GB of storage and 20 GB of RAM.

Any help would be greatly appreciated.  

![image](https://user-images.githubusercontent.com/32460863/112370348-80483480-8c9a-11eb-80e4-62e5d4329b22.png)
<img width="1702" alt="Capture" src="https://user-images.githubusercontent.com/32460863/112370394-8fc77d80-8c9a-11eb-8507-20030c77fecf.PNG">

**Comment by weslambert** [comment]
Have you tried using Virtualbox or VMWare Workstation and comparing the results?  You can also try switching to terminal and tailing the log in `/root/sosetup.log`.
